integration-tests/structured-logging/__tests__/ipc-send.js

```diff
@@ -0,0 +1,56 @@
+const { spawn } = require(`child_process`)
+const path = require(`path`)
+
+jest.setTimeout(20000) // 20s
+
+const gatsbyBin = path.join(
+  `node_modules`,
+  `gatsby`,
+  `dist`,
+  `bin`,
+  `gatsby.js`
+)
+
+describe(`IPC Send`, () => {
+  let gatsbyProcess
+
+  beforeAll(() => {
+    gatsbyProcess = spawn("node", [gatsbyBin, `develop`], {
+      stdio: [`ignore`, `ignore`, `ignore`, `ipc`],
+      env: {
+        ...process.env,
+        NODE_ENV: `development`,
+      },
+    })
+  })
+
+  afterAll(async () => {
+    gatsbyProcess.kill()
+  })
+
+  // TODO remove to a more global integration test but for now we want to make sure it gets an external send command
+  it(`should kill the develop process when exit command is given`, done => {
+    expect.assertions(1)
+    gatsbyProcess.on("exit", () => {
+      expect(true).toBe(true)
```

Comment:
true ...true ... ;)

please ignore this - I get why, just couldn't not comment on this :)